src/ParseObject.js

```diff
@@ -627,7 +631,7 @@ export default class ParseObject {
 
     // Validate changes
     var validation = this.validate(newValues);
```

Comment:
Thought: it might make more sense to wrap this entire `var validation = ...; if (...) {}` block in an
`if (!options.ignoreValidation)` statement. That way, we don't needlessly run the validation logic.